ups/eupspkg.cfg.sh

```diff
@@ -35,9 +35,38 @@ build()
     fi
 
 	detect_compiler
-	echo "Building boost with cxxflags=$CXX_CXX11_FLAG"
 
-	./b2 -j $NJOBS cxxflags=$CXX_CXX11_FLAG
+	#
+	# Try to find appropriate setting for CXX_CXX14_FLAG if not already set (to possibly empty value).
+	# This duplicates code recently added to detect_compiler in eupspkg.sh; it is here temporarily as a
+	# seatbelt to enable this build until we can be sure that eupspkg.sh has been updated from upstream.
+	#
+	# --** PLEASE REMOVE WHEN EUPS + EUPSPKG.SH ARE UPDATED FROM UPSTREAM **--
+	#
+
+	if [ -z "${CXX_CXX14_FLAG+1}" ] ; then
```

Comment:
Is the `+1` really needed?

Reply:
The `+1` is the magic that lets the conditional distinguish between unset and set to empty value.  This test is meant to be for CXX_CXX14_FLAG strictly unset.

Reply:
I was wondering if that was it. I don't see it much.